waspc/cli/Wasp/Cli/Command/Db/Migrate.hs

```diff
@@ -1,31 +1,25 @@
 module Wasp.Cli.Command.Db.Migrate
   ( migrateDev,
-    copyDbMigrationsDir,
-    MigrationDirCopyDirection (..),
   )
 where
 
-import Control.Monad.Catch (catch)
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 -- Wasp generator interface.
 
-import qualified Path as P
-import qualified Path.IO as PathIO
-import StrongPath (Abs, Dir, Path', (</>))
-import qualified StrongPath.Path as SP.Path
+import StrongPath ((</>))
 import Wasp.Cli.Command (Command, CommandError (..))
 import Wasp.Cli.Command.Common
   ( findWaspProjectRootDirFromCwd,
     waspSaysC,
   )
 import qualified Wasp.Cli.Common as Cli.Common
 import Wasp.Cli.Terminal (asWaspFailureMessage, asWaspStartMessage, asWaspSuccessMessage)
-import Wasp.Common (WaspProjectDir, dbMigrationsDirInWaspProjectDir)
-import Wasp.Generator.Common (ProjectRootDir)
-import Wasp.Generator.DbGenerator (dbMigrationsDirInDbRootDir, dbRootDirInProjectRootDir)
 import qualified Wasp.Generator.DbGenerator.Operations as DbOps
 
+-- | Note(shayne): Migrates using migrations dir in project dir. The source down to project
```

Comment:
```suggestion
-- | Note(shayne): Migrates using migrations dir in generated project dir. The source down to project
```
This is correct, right? Just checking because wasn't sure myself :D

Reply:
Yes! The naming is terrible and my fault -> I called them "wasp dir" and "project dir" and now it is very confusing. We should rather call them "wasp project dir" and "generated project dir" or something even better. Shayne and I opened an issue for it recently so I hope we will improve that soon. In the meantime I think it is good to add "generated" here as you did.